bamboo/docker-compose-local.yml

```diff
@@ -22,5 +19,6 @@ services:
       - 3030:3030
       - 2222:2222
       - 4550-4599:4550-4599
+      - 5432:5432
```

Comment:
This is a minor fix to the local unit tests - an update to the CI stack properly set the postgres container to have an internal port, but broke the local unit configuration.    Adding here to support blocked tickets only.